pallets/pips/src/lib.rs

```diff
@@ -1314,13 +1204,11 @@ impl<T: Trait> Module<T> {
     /// Remove the PIP with `id` from the snapshot if it is there.
     fn maybe_unsnapshot_pip(id: PipId, state: ProposalState) {
         if let ProposalState::Pending = state {
-            let cool_until = Self::proposals(id).unwrap().cool_off_until;
-            if cool_until <= <system::Module<T>>::block_number()
-                && <SnapshotMeta<T>>::get()
-                    .filter(|m| cool_until <= m.created_at)
-                    .is_some()
-            {
-                // Proposal is pending, no longer in cool-down, and wasn't when snapshot was made.
+            // Pending so therefore in live queue; evict `id`.
```

Comment:
Given the current use of this function, not sure we really need the `maybe` bit - i.e. it is only called w/ a Pending state now.

Reply:
It can also be called when `Scheduled`.